cms/static/cms/sass/components/_toolbar.scss

```diff
@@ -241,20 +241,14 @@
 
     .cms-toolbar-item {
         float: none;
-        [dir=rtl] & {
```

Comment:
As the comments say, if you didn't set `float` to `none`, if you're on mobile, the `cms-toolbar-item` will float to the right and won't be clickable:
![image](https://github.com/django-cms/django-cms/assets/50373064/01b9423f-38e8-432e-8602-4756b49874af)